user-interface/src/login/Login.tsx

```diff
@@ -56,7 +59,12 @@ export function Login(props: LoginProps): React.ReactNode {
       break;
     case 'none':
       providerComponent = (
-        <Session provider={provider} user={props.user ?? { name: 'Super User' }}>
+        <Session
```

Comment:
Should we continue to support the "none" option? If so we should upgrade it to use the oauth2 endpoint to gen a valid JWT.

Reply:
the none option is used by pa11y testing at the moment

Reply:
Fair enough.